srv/modules/runners/ui_rgw.py

```diff
@@ -122,7 +122,15 @@ def endpoints(cluster='ceph'):
         port = '7480'  # civetweb default port
         ssl = ''
         admin_path = 'admin'
-        for rgw_conf_file_path in glob.glob("/srv/salt/ceph/configuration/files/ceph.conf.*"):
+
+        rgw_names = ['rgw']
+        if 'roles' in __pillar__:
+            if 'rgw_configurations' in __pillar__:
```

Comment:
so this doesn't work in a runner, `__pillar__` is always empty only works on salt modules, need to use local.cmd or the pillar util from above